x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/columns/report_definition_col.tsx

```diff
@@ -69,21 +66,20 @@ export function ReportDefinitionCol({
         <DatePickerCol seriesId={seriesId} />
       </EuiFlexItem>
       <EuiHorizontalRule margin="xs" />
-      {indexPattern &&
-        reportDefinitions.map(({ field, custom, options }) => (
-          <EuiFlexItem key={field}>
-            {!custom ? (
-              <ReportDefinitionField
-                seriesId={seriesId}
-                dataSeries={dataViewSeries}
-                field={field}
-                onChange={onChange}
-              />
-            ) : (
-              <CustomReportField field={field} options={options} seriesId={seriesId} />
-            )}
-          </EuiFlexItem>
-        ))}
+      {reportDefinitions.map(({ field, custom, options }) => (
+        <EuiFlexItem key={field}>
+          {!custom ? (
```

Comment:
Just curious, I know this wasn't added in this PR but just trying to understand the feature. Why do we need the option to have a custom report field? What is an example of a report that needed a custom field and why?